tensorrt_llm/_torch/attention_backend/sparse/qsa/module.py

```
@@ -0,0 +1,219 @@
+                request_indices=req_idx,
+                metadata=attn_metadata,
+                softmax_scale=1.0 / (attention.q_scaling * attention.head_dim**0.5),
+            )
+            return output.reshape(num_tokens, -1)
+
+        output = torch.empty_like(q)
+        seq_lens = attn_metadata.seq_lens[: attn_metadata.num_seqs].tolist()
+        kv_lens = attn_metadata.kv_lens_runtime.tolist()
+        token_offset = 0
+        chunk_size = _query_chunk_size()
+        for request_idx, (query_len, sequence_len) in enumerate(zip(seq_lens, kv_lens)):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)